recipe-client-addon/lib/StudyStorage.jsm

```diff
@@ -0,0 +1,108 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * @typedef {Object} Study
+ * @property {string} name
+ *   Unique name of the study
+ * @property {number} version
```

Comment:
This is documented as the add-on version number, which would be a string as @gijsk suggests. I wasn't thinking of storing the revision number of the recipe as well, but that might be useful info to have as well (although how we'd expose it, I don't know). If we did, it'd be a different field. We store revision numbers on the backends as strings too, though.

Let's make this a string, and consider whether we want to send and store the recipe revision number as well in a separate issue.